integration-tests/test/privateDataExchange.ts

```diff
@@ -59,7 +53,7 @@ let exchangeData: any = {};
 const preparePassport = async () => {
 
   // Accounts
-  monethaOwner = await getAccount(web3, 0);
+  monethaOwner = await getAccount(web3, isQuorum ? 0 : 5);
```

Comment:
So we assume that `privateDataExchange` tests will run after previous tests - so we use other account (nr 5) to prevent nonce issue - right?

This will break very quickly when I add new tests and they get somewhere in -between. Unfortunatelly, this solution is not enough for us

Reply:
Also - task was registered to solve problem with Pantheon, and not Quorum (even though it may have the same problem). I see no code changes for Pantheon